lab-max/lib/parse-body.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = function(req, callback){
+  req.body = '';
+  req.on('data', function(data){
+    req.body += data.toString();
+  });
+
+  req.on('end', function(){
+    try {
+      req.body = JSON.parse(req.body);
+      callback(null, req.body);
```

Comment:
@meeksfred good callback structure - naming conventions conform to industry standards